assignments/closure.js

```diff
@@ -1,21 +1,81 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const add = (a, b) => {
+  const c = 12;
+  return a + b + c;
```

Comment:
It doesn't look like you have a closure here. You would need to write another function inside of add that could access c. That second function would be the closure. So, it could look something like this:
```suggestion
const add = (a, b) => {
  const c = 12;
  function total() {
    return a + b + c
  }
  return total()
}
```